plutus-tutorial/doc/intro.adoc

```diff
@@ -29,3 +29,14 @@ authors do not write Plutus Core directly. The Plutus Platform is a
 software development kit to enable smart contract authors to easily
 write smart contracts, including the logic that will eventually be run
 on the blockchain as Plutus Core.
+
+To code a smart contract in Plutus, you can code directly in the Plutus 
+Playground editor, or you can use your preferred code editor and save the 
```

Comment:
This is the first mention of Plutus Playground in the document, it probably needs an introduction. And you probably want to link to it - there's an attribute for the url, see other places in the document where we link to it.

Reply:
will include description and link